systemd/openqa-worker-cacheservice.service

```diff
@@ -2,6 +2,7 @@
 Description=OpenQA Worker Cache Service
 After=network-online.target
 Wants=network-online.target
+PartOf=openqa-worker.target
```

Comment:
I wonder if the cache-service can be set to start _before_ the workers are available (Call me paranoid :P)

Reply:
Hups. Thought "resolving" it will also post my comment. What I wrote:
One could add this for sure but this change is only to be able to type `systemctl stop openqa-worker.target` and stop everything what is openQA/worker related.